templates/mod/mime.conf.erb

```diff
@@ -31,6 +31,8 @@ AddLanguage sv .sv
 AddLanguage zh-CN .zh-cn
 AddLanguage zh-TW .zh-tw
 
-AddHandler type-map var
-AddType text/html .shtml
-AddOutputFilter INCLUDES .shtml
+<%- @mime_types_additional.sort.each do |key, value_hash| -%>
+        <%- value_hash.each do |value_hash_key, value_hash_value| %>
+<%=key%> <%=value_hash_key%> <%=value_hash_value-%>
```

Comment:
```erb
<%= add_mime %> <%= type %> <%= extension%>
```
also note: i'm removing the `-` from the `%>`, or else we'll get all of those AddType (etc) config lines in one line.